assets/dev/js/editor/controls/media.js

```diff
@@ -222,27 +255,78 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		}
 
 		if ( attachment.url ) {
-			this.setValue( {
+			const updatedState = {
```

Comment:
Why?

Reply:
I save the control state on every change, So when the user switch from one breakpoint to another, I can get the placeholders from the preview's breakpoint. And show them as defaults.